mediaportal/MediaPortal.Base/skin/Titan/mytvrecordedtv.xml

```diff
@@ -6,7 +6,6 @@
 <define>#header.label:604</define>
 <controls>
 
-    <import>common.default.background.xml</import> 
```

Comment:
Don't we need background here?

Reply:
Probably, I just copied this over from MePo but did not check it. I personally have not made any changes to this as I don't use the plugin.